README.md

```diff
@@ -43,6 +43,19 @@ already have these, if not please download and run vcredist.x64.exe from
 [Microsoft](https://support.microsoft.com/en-gb/help/2977003/the-latest-supported-visual-c-downloads) to install them prior to running
 cbmc.
 
+### macOS
+
+For macOS there is a [Homebrew](https://github.com/Homebrew/brew) package available. Once you have installed Homebrew,
```

Comment:
I think the Homebrew link should be `https://brew.sh` rather than the github link. brew.sh is the "user" webpage, and importantly, says up-front how to install Homebrew :-)

It might also be worth linking the word 'available' to `https://formulae.brew.sh/formula/cbmc` ?